### ISSUE TITLE
Unable to focus search input with keyboard in Filter Modal

### ISSUE THREAD

**Issue by kamilmielnik** [metabase/metabase#52733]
### Describe the bug

https://github.com/user-attachments/assets/994e7f38-7601-456c-82ab-945f7a0a59f2

### To Reproduce

1. New > Question > Orders > Visualize
2. Click "Filter"
3. Try to use <kbd>Tab</kbd> to focus the search input

### Information about your Metabase installation

master, a7b8f79b031eefeff5b3b0c8f8f40bcb7fd2e005

### Severity

P3
